middleware/src/handlers/handlers.go

```diff
@@ -0,0 +1,87 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+	//"encoding/json"
```

Comment:
Nit: Could we remove unused dependencies entirely, instead of commenting them out?

(If there's future work where we expect to add them, we could keep them, but if so I'd add `TODO` comments explaining what's planned.)

(And elsewhere.)

Reply:
`golangci-lint run` now detects this.